content/en/faqitems/security-1-immutability-files-change-on-host.markdown

```diff
@@ -0,0 +1,14 @@
++++
+question = "Does Bottlerocket’s immutability mean that no files will ever change on the host?"
+group = "Security"
++++
+
+There are both immutable and mutable areas on a Bottlerocket host.
```

Comment:
You don't directly answer the question if you keep it as yes/no.

Reply:
Rephrasing the question to: "I heard Bottlerocket is immutable. What does that really mean?"